packages/@lwc/scoped-registry/package.json

```diff
@@ -0,0 +1,51 @@
+{
+    "name": "@lwc/scoped-registry",
+    "version": "2.25.1",
+    "description": "Define different custom elements sharing the same tag name",
+    "homepage": "https://lwc.dev/",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/salesforce/lwc.git",
+        "directory": "packages/@lwc/scoped-registry"
+    },
+    "bugs": {
+        "url": "https://github.com/salesforce/lwc/issues"
+    },
+    "main": "dist/index.cjs.js",
+    "module": "dist/index.js",
+    "license": "MIT",
+    "typings": "types/index.d.ts",
+    "scripts": {
+        "clean": "rm -rf dist/ types/",
+        "build": "rollup --config scripts/rollup.config.js",
+        "dev": "rollup  --config scripts/rollup.config.js --watch --no-watch.clearScreen"
+    },
+    "files": [
+        "dist/",
+        "types/"
+    ],
+    "keywords": [
+        "lwc",
+        "polyfill",
+        "custom",
+        "elements",
+        "scoped",
+        "registry"
+    ],
+    "dependencies": {
+        "@lwc/shared": "2.25.1"
+    },
```

Comment:
`@lwc/shared` must be a dependency, not a `devDependency`, so that is correctly de-duplicated in `@lwc/engine-dom`. This is similar to how `@lwc/aria-reflection-polyfill` works.